README.md

```diff
@@ -21,7 +21,6 @@ that the text following the # characters are comments and will not appear in the
 ```yaml
 serviceName: Discounted Icecreams       # The service name that will appear in the title of the accessibility statement.
                                         # Do not include the word service at the end, as this will be added by the templates
-serviceHeaderName: Icecreams            # The service name as it should appear in the grey Gov.UK header bar
```

Comment:
Suggestion: shall we also remove this parameter from our examples YAML files?
- example-automated-testing-only.yml
- example-fully-compliant.cy.yml
- example-fully-compliant.yml
- example-non-compliant.yml
- example-partially-compliant.cy.yml
- example-partially-compliant.yml

Reply:
Good suggestion, yes I think so.